tests/test_chatbot_output.py

```diff
@@ -207,4 +224,3 @@ def test_database_is_not_updated_when_read_only(self):
 
         self.assertFalse(exists_before)
         self.assertFalse(exists_after)
-
```

Comment:
This looks like Atom was removing the extra blank line at the end of the file.

There were two blank lines before, now there is only one.